ape_safe/client/mock.py

```diff
@@ -111,5 +111,5 @@ def post_signatures(
             )
 
     def estimate_gas_cost(self, receiver: AddressType, value: int, data: bytes, operation: int = 0):
-        # TODO
-        return 0
+        # Estimate normally.
+        return NotImplemented
```

Comment:
Wait, I don't think returning an error is the right way to go here

Reply:
It is a constant, not an error, but feel free to change.
https://stackoverflow.com/questions/1062096/python-notimplemented-constant

Reply:
alright alright we'll just have it make the request normally or something.
The weird part is that Safe's API is like different a bit than just normal eth rpc stuff